### ISSUE TITLE
Publish signed Podman*.msi

### ISSUE THREAD

**Issue by naiksujit** [containers/podman#15973]
Many organization blocks non signed applications, including mine.

It would be great if the Podman.msi is signed and published.

I have checked recent released version and that is not signed.

[v4.2.1](https://github.com/containers/podman/releases/tag/v4.2.1)

**Comment by ArvindGundimajalu** [comment]
Yup, this will really help me absorb the product into my organization too.

**Comment by Luap99** [comment]
@n1hility @gbraad PTAL
